tests/Unit/Helpers/PointwiseFunctions/AnalyticSolutions/FirstOrderEllipticSolutionsTestHelpers.hpp

```diff
@@ -80,6 +78,8 @@ void verify_solution(
             solution_fields, fluxes, expanded_sources_args...);
       },
       sources_args);
+  auto div_fluxes = divergence(std::move(fluxes), mesh,
+                               coord_map.inv_jacobian(logical_coords));
```

Comment:
`divergence` takes its argument by reference, so the `std::move` shouldn't be there at all.